ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -184,6 +184,10 @@ private void updateCnecExtensions(Crac crac, String resultVariantId, SystematicS
             CnecResult cnecResult = cnecResultMap.getVariant(resultVariantId);
             cnecResult.setFlowInMW(systematicSensitivityAnalysisResult.getCnecFlowMap().getOrDefault(cnec, Double.NaN));
             cnecResult.setFlowInA(systematicSensitivityAnalysisResult.getCnecIntensityMap().getOrDefault(cnec, Double.NaN));
+            cnecResult.setMinThresholdInMW(cnec.getMinThreshold(Unit.MEGAWATT).orElse(Double.NEGATIVE_INFINITY));
+            cnecResult.setMaxThresholdInMW(cnec.getMaxThreshold(Unit.MEGAWATT).orElse(Double.POSITIVE_INFINITY));
+            cnecResult.setMinThresholdInA(cnec.getMinThreshold(Unit.AMPERE).orElse(Double.NEGATIVE_INFINITY));
+            cnecResult.setMaxThresholdInA(cnec.getMaxThreshold(Unit.AMPERE).orElse(Double.POSITIVE_INFINITY));
```

Comment:
Why not a method that get a Cnec in argument and perform all these set methods directly in CnecResult ?

Reply:
true, I'll change that